docker-compose.ci.yml

```diff
@@ -24,12 +25,21 @@ services:
       - mysql_vol:/var/lib/mysql_tmpfs/
       - .:/obs
     command: /obs/contrib/start_test_db
-  backend:
+  tbackend:
```

Comment:
[nitpick] 
I don't like the naming of tbackend and backend ....

Reply:
I don't even - I just needed to free the 'backend' name and did a quick shot.